cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -487,7 +489,12 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     this.$('input.practice_exam').prop('checked', true);
                 } else {
                     this.$('input.proctored_exam').prop('checked', true);
-                    this.$('.field-exam-review-rules').show();
+                    let showReviewBlock = this.model.get('show_review_rules');
+                    if (!showReviewBlock) {
+                      this.$('.field-exam-review-rules').hide();
```

Comment:
Is the review rules section hidden by default? Based on the previous code, it looks like it's not necessary to explicitly hide it here, so you could eliminate this condition.

Reply:
The existing code is needed to show the section when a proctored exam is selected and hide it if a timed exam is selected. The point of waffle flag, is to hide it even when a proctored exam is selected if the flag to show it is not set. Therefore we need both conditionals.